pkg/ipc/ipc.go

```diff
@@ -493,13 +493,17 @@ func readUint32Array(outp *[]byte, size uint32) ([]uint32, bool) {
 	if int(size)*4 > len(out) {
 		return nil, false
 	}
+	// "Convert" the data to uint32.
```

Comment:
Please send this commit as a separate PR as well.

Reply:
Done: https://github.com/google/syzkaller/pull/4515